app/views/users/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="clearance" class="sign-up">
+  <h2><%= t(".title") %></h2>
+  <%= form_for @user do |form| %>
+    <%= render partial: "/users/form", object: form %>
```

Comment:
What does this object argument do? Does it somehow pass the `form` object through with the name `form` for the partial?